### ISSUE TITLE
Bug in get_vs_prefix() 

### ISSUE THREAD

**Issue by pat357** [jb-alvarado/media-autobuild_suite#251]
There is a minor bug in this function:
```
    elif regtool check "$regkey" >/dev/null 2>&1; then
        # last resort, check registry for install
        vsprefix=$(regtool get "$regkey")
-        echo $(cygpath -u "${vsprefix%/}")  <-- delete
+       echo $(cygpath -u "${vsprefix%\\}")  <-- add 
```
"${vsprefix%/}" should remove the trailing /, but there are no / in a Windows path.
The replacement line with "${vsprefix%\\}") will remove the trailing \

**Comment by pat357** [comment]
BTW is there a way to search for my comments I gave on your changes ?

Again, I've forgotten where I posted my question about the difference between var[#] and var[@].
When should I use which one ?
What is the difference beween the 2 ?

**Comment by wiiaboo** [comment]
It's the third time I have to search for that. Just look at the list of commits and open the ones with comments.

**Comment by pat357** [comment]
```
Let me rephrase my Q :+1: 
I mean the difference between var[@] and var[*].

If var is an array (1 dimensional , n elements) ;  are both the [@] and [*] valid to select ALL elements from this array ? 
When should I use which one ?

**Comment by wiiaboo** [comment]
?

**Comment by wiiaboo** [comment]
http://stackoverflow.com/a/3355375
